.changeset/clever-ghosts-roll.md

```diff
@@ -0,0 +1,7 @@
+---
+"@vanilla-extract/esbuild-plugin": minor
+"@vanilla-extract/integration": minor
```

Comment:
This is actually a major for the `integration` package as it contains a breaking change for how you pass externals.